clientv3/client.go

```diff
@@ -191,7 +202,7 @@ func (c *Client) ActiveConnection() *grpc.ClientConn {
 }
 
 // retryConnection establishes a new connection
-func (c *Client) retryConnection(oldConn *grpc.ClientConn, err error) (*grpc.ClientConn, error) {
+func (c *Client) retryConnection(err error) (newConn *grpc.ClientConn, dialErr error) {
```

Comment:
this method name and `connRetry` are too similar I feel. probably we can change the connRetry to requestConnRetry? basically connRetry does not actually retry on the connection but ask the monitor loop to do it.